notebooks/textbook/Quantum_Neuron/Quantum_Neuron.ipynb

```diff
@@ -0,0 +1,717 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantum Neuron\n",
+    "\n",
+    "In this work, we build [quantum neuron](https://arxiv.org/abs/1711.11240) with [PennyLane](https://pennylane.ai/), and run it on different devices such as SV1, IonQ, and Aspen-M-2.\n",
```

Comment:
Aspen M-2 is now retired as of May 02. Let's change Aspen M-2 to M-3 wherever applicable.

Reply:
Done.